pkgs/applications/misc/calibre/default.nix

```diff
@@ -169,6 +169,7 @@ stdenv.mkDerivation rec {
 
   # Wrap manually
   dontWrapQtApps = true;
```

Comment:
```suggestion
```

Reply:
this is necessary.  If removed the following error appears:
```
qmakePrePhase
qtPreHook
Error: wrapQtAppsHook is not used, and dontWrapQtApps is not set.
```